tests/utils/test_ddcHelpers.cpp

```diff
@@ -84,9 +90,9 @@ TEST(DDCHelper, NonUniformPeriodicRestriction)
     }
     nu_points[5] = x_max;
 
-    ddc::init_discrete_space<GridNonUniform>(nu_points);
+    ddc::init_discrete_space<Grid1D>(nu_points);
```

Comment:
Better to use points that are actually non-unfiorm
```suggestion
    ddc::init_discrete_space<Grid1D>(build_random_non_uniform_break_points(x_min, x_max, ncells));
```
(I think this test was written before `build_random_non_uniform_break_points`